### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by redwing555**
In this project the following requirements have been implemented : 

- Create a new Rails app called 'hello-rails-react'.
- Set up the project with `webpacker` the `react-rails` gems.
- Use a Postgres database set up.
- Create a table for storing the messages and create and seed 5 different greetings.
- Create an API endpoint that selects a random greeting from the messages table.
- Create a static view and set it to be the root of the app.
- Create the App component with react-router and render it in the static view.
- Create the Greeting component that will display a greeting. Set up a route for it in the App component.
- Create a store, an action, and a reducer using `Redux` that will connect to the API endpoint to get the random greeting.
- Display the random greeting in the Greeting component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,26 @@
...
+ # Hello-Rails-React
+
+This is the first trial of  working with  React as a front-end and Ruby-on-Rails as a backend. Random messages are displayed on button click.
+## Built With
+
+- Ruby On Rails
+- React-Rails gem
+- Redux
+- Git
+
```
Comment (elisha2kyakpo1) [inline README.md:10]: - [x] The readme file is missing some important information. Let's include information on how a user can get a copy of the project and running it locally.

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import configureStore from '../redux/configureStore';
+import Greeting from './Greeting';
+
+const store = configureStore();
+class App extends React.Component {
```
Comment (elisha2kyakpo1) [inline app/javascript/components/App.js]: - [x] OPTIONAL --- We should prefer functional components to class based components.

File: app/javascript/components/Greeting.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { createStructuredSelector } from 'reselect';
+import { connect } from 'react-redux';
+import './greeting.css';
+
+const GET_GREETING_SUCCESS = 'GET_GREETING_SUCCESS';
+const GET_GREETING_REQUEST = 'GET_GREETING_REQUEST';
+
+function getGreeting() {
```
Comment (elisha2kyakpo1) [inline app/javascript/components/Greeting.js]: - [x] Let's use ES6 syntax for all our functions.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi @redwing555,

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
- The project title and PR description looks professional ✔️
- The project works as expected ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- `console.log` statements (code) should only be used when debugging

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @redwing555, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS APPROVED ✅ 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**